### DISCUSSION TITLE
스크린샷 모드에서제스처 기능 질문입니다!!

### DISCUSSION THREAD

**Discussion by Park-sechang** [naverz/zepeto-studio-kor#448] [5. Q&A]
제스처를 추가하고 싶은데. getureListButtons 와 gestureClips를 추가해도 게임을 실행하면 3개로 됩니다. 

GestureController 스크립트에서 개수를 조절하지 않는거 같고,

다른 스크립트에서 조절하고 있는거 같은데. 어디에서 조절하는지 잘 모르겠네요.

제스처를 어떻게 추가하는지 알 수 있을까요??

![Screenshot_6](https://user-images.githubusercontent.com/99107908/156273120-888a94de-b419-437e-b568-1ac1a417b660.png)

**Comment by pumfit** [comment]
아래와 같이 모듈의 프리팹안에서 제스처 버튼을 생성하고 제스처컨트롤러에 할당해주시면 됩니다!
1. 먼저 수정을 위해서 ZepetoScreenShot모듈의 프리팹을 열어주시고
![image](https://user-images.githubusercontent.com/46295539/156278725-e7d918df-b233-499b-beef-c449efa676dd.png)
2. Hierarchy 창에서 ZepetoScreenShotCanvas - GestureListView - ... - Content에서 버튼을 새로 추가해줍니다.
![image](https://user-images.githubusercontent.com/46295539/156279229-fadcd81b-de00-4e52-85cd-1cc9ef3253ef.png)
3. ZepetoGestureContorller에 버튼 배열에 추가한 버튼을 넣어주고 추기할 제스처도 추가해줍니다.
![image](https://user-images.githubusercontent.com/46295539/156280069-5e579910-f4a1-4b34-a1d3-e98ca86c33f7.png)
4.프리팹을 저장하고 플레이를 누르고 추가한 버튼을 클릭하면 제스처가 재생되는 것을 볼 수 있습니다.
